samwise.js

```diff
@@ -2,11 +2,10 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.